pkg/storage/couchdb/couchdbstore.go

```diff
@@ -155,38 +154,30 @@ func (c *CouchDBStore) Put(k string, v []byte) error {
 		return errors.New("key and value are mandatory")
 	}
 
-	var valueToPut []byte
-	if isJSON(v) {
-		valueToPut = v
-	} else {
-		valueToPut = wrapTextAsCouchDBAttachment(v)
-	}
-
 	revID, err := c.getRevID(k)
 	if err != nil {
 		return err
 	}
 
+	doc := map[string]interface{}{"payload": v}
```

Comment:
you can put all data in one field

Reply:
you need to check data type

Reply:
we do not need to, it is always `[]byte`

Reply:
@fqutishat  why I cannot put data to one field? it is always `[]byte` I put `[]byte` I get `[]byte`.
It solves the problem and it even improves performance (less Marshall/Unmarshall).

Reply:
it's not good if data is big

Reply:
in some cases you need to save big binary in attachment like images

Reply:
@troyronda

Reply:
ok, I will combine it

Reply:
@soluchok please put a comment in there with the rationale, much appreciated :)

Reply:
I rollbacked changes. But I wrapped JSON payload with the `payload` field.
@llorllale the problem was that we couldn't use keys such as "_key, _rev, etc."